test/fakeintake/go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/benbjohnson/clock v1.3.5
 	github.com/cenkalti/backoff/v4 v4.2.1
 	github.com/kr/pretty v0.3.1
+	github.com/mattn/go-sqlite3 v1.14.22
```

Comment:
You cannot use `github.com/mattn/go-sqlite3`, it's a CGO package, we don't want CGO in the FI as it implies it would not be portable anymore.